### PR TITLE
Feat/mentor set assignment

### PR DESCRIPTION
**Pull request by abrahammegantoro**
## Story/Task

[_Link ke story/task di Trello_](https://trello.com/c/P1EthLl1)

## Summary

Add setAssignmentScore

## Screenshot

image

### CODE REVIEW

File: src/server/api/routers/assignment.ts [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+    .mutation(async ({ ctx, input }) => {
+      const { submissionId, score } = input;
+
+      const submission = await ctx.prisma.assignmentSubmission.findUnique({
+        where: { id: submissionId }
+      });
+
+      if (!submission) {
+        throw new Error('Submission not found');
+      }
```
Comment (msyahrulsp) [inline src/server/api/routers/assignment.ts:48]: Ganti jadi `TRPCError`

Comment (msyahrulsp) [inline src/server/api/routers/assignment.ts:48]: Ganti jadi `TRPCError`

File: src/server/api/routers/assignment.ts [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+        limitPerPage: z.number()
+      })
+    )
+    .query(async ({ ctx, input }) => {
+      return await ctx.prisma.assignment.findMany({
+        where: {
+          title: {
+            contains: input.searchQuery
+          }
+        },
```
Comment (msyahrulsp) [inline src/server/api/routers/assignment.ts:26]: Ini filterby nya ga dipake kah?

**Review by msyahrulsp — CHANGES_REQUESTED** [review]
Tolong diubah ya

File: src/server/api/routers/assignment.ts (hunk 2 above)
Comment (abrahammegantoro) [reply src/server/api/routers/assignment.ts:26]: Sorry kak, yang procedure getAssignnment bukan bagianku

**Comment by msyahrulsp** [comment]
Ini branchnya ubah dlu ke staging

**Pull request closed without merging**